graylog2-server/src/main/java/org/graylog/events/rest/EventDefinitionsResource.java

```diff
@@ -314,19 +318,61 @@ public Response bulkDelete(@ApiParam(name = "Entities to remove", required = tru
     @Consumes(MediaType.WILDCARD)
     @ApiOperation("Enable event definition")
     @AuditEvent(type = EventsAuditEventTypes.EVENT_DEFINITION_UPDATE)
-    public void schedule(@ApiParam(name = "definitionId") @PathParam("definitionId") @NotBlank String definitionId) {
+    public EventDefinitionDto schedule(@ApiParam(name = "definitionId") @PathParam("definitionId") @NotBlank String definitionId,
+                                       @Context UserContext userContext) {
         checkPermission(RestPermissions.EVENT_DEFINITIONS_EDIT, definitionId);
+        final Optional<EventDefinitionDto> eventDefinitionDto = dbService.get(definitionId);
         eventDefinitionHandler.schedule(definitionId);
+        return eventDefinitionDto.orElse(null);
+    }
+
+    @POST
+    @Path("/bulk_schedule")
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Timed
+    @ApiOperation(value = "Enable a bulk of event definitions", response = BulkOperationResponse.class)
```

Comment:
i used `bulk` to makes this consistent with `bulk_delete` used here and in streams.

Reply:
That's fine. I just mean the string `"Enable a bulk of event definitions"`. Bulk as a noun refers to a [homogeneous mass](https://www.merriam-webster.com/dictionary/bulk#dictionary-entry-1), rather than a grouping of distinct elements. 
If we want to stick with it, then I prefer e.g. `"Enable event definitions in bulk"`